Packs/SkyboxSecurity/Integrations/SkyboxTicketsService/README.md

```diff
@@ -0,0 +1,5947 @@
+
+
+### skybox-setTicketFields
+***
+Sets ticket data in Skybox.
+You can use this method with all ticket types.
+
+
+#### Base Command
+
+`skybox-setTicketFields`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| ticketId | The ID of the ticket. | Optional | 
+| ticketIdType | Specifies whether the ticket ID is the Skybox ticket ID or the ID from the<br/>external ticketing system<br/>Possible values:<br/>l SBV<br/>l EXTERNAL. Possible values are: SBV, EXTERNAL. | Required | 
+| ticketFields_typeCode | The Code type of the field. | Optional | 
+| ticketFields_value | The field value. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Skybox.setTicketFields | String | The ticket fields that have been set | 
+
+
+#### Command Example
+``` ```
+
+#### Human Readable Output
```

Comment:
Please fill in the empty sections or remove the blank titles.
Please go through the entire file and look for more such sections.

Reply:
fixed